homeassistant/components/zeversolar_local/sensor.py

```diff
@@ -0,0 +1,78 @@
+"""Zever solar sensors."""
+import logging
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_POWER,
+    STATE_CLASS_MEASUREMENT,
+    STATE_CLASS_TOTAL_INCREASING,
+    SensorEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ENERGY_KILO_WATT_HOUR, POWER_WATT
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import COORDINATOR, DOMAIN, ZEVER_INVERTER_ID
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+):
+    """Set up the hunter douglas shades sensors."""
+
+    data = hass.data[DOMAIN][entry.entry_id]
+    coordinator = data[COORDINATOR]
+    _LOGGER.debug("Setting up zever solar sensors")
```

Comment:
This is already logged as `INFO (MainThread) [homeassistant.components.sensor] Setting up  sensor.zeversolar_local` in:
https://github.com/home-assistant/core/blob/27ecd43da36da22f68e74cfab977c310d720aafd/homeassistant/helpers/entity_platform.py#L235
```suggestion
```